spec/index.bs

```diff
@@ -2641,12 +2641,12 @@ Using external textures costs several binding points: see [=binding cost=].
     The configuration used may not be stable across time, systems, user agents, media sources,
     or frames within a single video source.
     In order to account for many possible representations,
-    the [=binding cost=] conservatively requires:
+    the [=binding cost=] conservatively requires, for *each* external texture:
 
-    - for pipeline layouts that have *any* external textures, one sampler binding
-        for sampling 3D LUTs.
-    - for *each* external texture, one sampled texture binding for each plane,
-        one sampled texture binding for a 3D LUT, and one uniform buffer binding for metadata.
+    - three sampled texture bindings (for up to 3 planes),
+    - one sampled texture binding for a 3D LUT,
+    - one sampler binding to sample the 3D LUT, and
+    - one uniform buffer binding for metadata.
```

Comment:
are we sure we wouldn't want this to have a dynamic offset?

Reply:
The buffer could be a single allocation at `GPUExternalTexture` creation time. It's probably cheap enough that we don't need a dynamic offset.

Reply:
It doesn't even have to be a single allocation, it just has to be at a non-dynamic offset (known at bind group creation).